tests/test_D1_model_components.py

```diff
@@ -105,24 +108,30 @@ def helper_test_transformer_in_model_and_dict(self, optimize, dict_asset):
         # self.models should contain the transformer (indirectly tested)
         # check output bus (`nominal_value`, `investment` and `existing`) these
         # values are expected to be different depending on whether capacity is optimized or not
-        if optimize is True:
-            output_bus = self.model.entities[-1].outputs.data[
-                self.busses[dict_asset[OUTPUT_BUS_NAME]]
-            ]
-            assert isinstance(
-                output_bus.investment, solph.options.Investment
-            )  # todo maybe ep costs
-            assert output_bus.investment.existing == dict_asset[INSTALLED_CAP][VALUE]
-            assert output_bus.nominal_value is None
-        elif optimize is False:
-            output_bus = self.model.entities[-1].outputs.data[
-                self.busses[dict_asset[OUTPUT_BUS_NAME]]
+        if multiple_outputs == True:
+            output_bus_list = [
+                self.model.entities[-1].outputs.data[self.busses[bus_name]]
+                for bus_name in dict_asset[OUTPUT_BUS_NAME]
```

Comment:
What happens in this for-loop?

Reply:
It's a list comprehension, the last "]" isn't shown in this patch here. It's: 
```
output_bus_list = [
                self.model.entities[-1].outputs.data[self.busses[bus_name]]
                for bus_name in dict_asset[OUTPUT_BUS_NAME]
            ]
```

So if there are multiple output busses, they are added to `output_bus_list` to be tested separately below.